components/Card/Card.module.css

```
@@ -0,0 +1,44 @@
+.card {
+  position: relative;
+  display: grid;
+  grid-gap: 12px;
```

Константы отступов есть в рутовых переменных
```suggestion
  grid-gap: var(--space-s);
```

---

И не только отступов, по возможности все константы оттуда бери или добавляй (пока основу делаем), за исключением, наверное, только ширины бордеров